public/index.html

```diff
@@ -87,7 +87,7 @@
     <% if (process.env.REACT_APP_INCLUDE_GTAG === 'true') { %>
     <script
       async
-      src="https://www.googletagmanager.com/gtag/js?id=G-H3S0KQSBGX"
+      src="https://www.googletagmanager.com/gtag/js?id=UA-387204-13"
```

Comment:
What's going on here? Which one is actually being used?

Reply:
It was an experimental checking the latest [Analytics 4](https://analytics.google.com/analytics/web/?hl=en#/p252327942/reports/defaulthome?params=_u..nav%3Ddefault) data.. it's a bit confusing and I think the old look is better.

Reply:
Adding those events to the same analytics account that we have already.